docs/Developing-Wallace-(setup-guide).md

```diff
@@ -10,7 +10,7 @@ python --version
 ```
 If you do not have Python 2.7 installed, you can install it from the [Python website](https://www.python.org/downloads/).
 
-Or, if you use Homebrew:
+Or, if you use Homebrew (if you use Anaconda, install using `conda`, not Homebrew):
```

Comment:
Yes if you need a different version: 

paraphrasing http://conda.pydata.org/docs/py2or3.html#update-or-upgrade-python

`conda install python=3.5` will upgrade your python in whatever branch you are in to python3.5 (and will still allow you to create new environments with your previous python version) 

but more generally if you were to create a new conda environment with Python 2.7 using  

```
conda create -n py27 python=2.7
```

I'm pretty sure if you didn't have 2.7 already installed, it would then install it, though I can't actually find the documentation describing that and don't want to create a clean wipe of my system to test this.